github-actions/action.yml

```diff
@@ -18,36 +18,35 @@ inputs:
   exclude:
     description: |-
       A list of paths (one per line) to exclude from the generation. Glob
-      patterns are accepted.
-      For reference, Ruby's Dir.glob is used:
-        https://ruby-doc.org/core-2.6.3/Dir.html#method-c-glob
+      patterns are accepted. For reference, Ruby's Dir.glob is used:
+      https://ruby-doc.org/core-2.6.3/Dir.html#method-c-glob.
     required: false
     default: |-
       third-party/**/*
 outputs: {}
 runs:
   using: composite
   steps:
-  - run: |-
-      if [ -z '${{ github.head_ref }}' ]; then
-        >&2 echo "Not a pull request..."
-        exit 1
-      fi
-    shell: bash
   - run: ${{ github.action_path }}/generate.rb
     shell: bash
     env:
       EXCLUDE: ${{ inputs.exclude }}
   - run: |-
       set -o errexit -o nounset -o pipefail
-
       mapfile -d '' -t < <(git ls-files -z --deleted --modified --others **/*README.md)
-      if [ "${#MAPFILE[@]}" -gt 0 ]; then
+      echo "::set-output name=files::${MAPFILE[@]}"
+      echo "::set-output name=count::${#MAPFILE[@]}"
+    id: modified_files
+    shell: bash
+  - run: |-
```

Comment:
We can use the `if` on the step itself, rather than in shell. That way it's clearer that it's skipped.

Reply:
I tried. :(
`if` is not allowed in the composite actions.

Reply:
Oh right because this isn't actually the action pipeline, it's the action implementation. Bleh why are they different.

Reply:
Not sure... I'm still figuring out GitHub Actions.